esp-hal-common/src/system.rs

```diff
@@ -293,6 +293,41 @@ pub struct Dma {
     _private: (),
 }
 
+pub enum RadioPeripherals {
+    #[cfg(phy)]
+    Phy,
+    #[cfg(bt)]
+    Bt,
+    #[cfg(wifi)]
+    Wifi,
+    #[cfg(ieee802154)]
+    Ieee802154,
+}
+
+pub struct RadioClockControl {
+    _private: (),
+}
+
+/// Control the radio peripheral clocks
+pub trait RadioClockContoller {
```

Comment:
`RadioClockContoller` should be `RadioClockController`

Reply:
Oh thanks! Took me a few minutes of reading this again and again until I realized what I mistyped 😄